sdk/core/System.BinaryData/src/BinaryData.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    /// <summary>
+    /// A lightweight abstraction for a payload of bytes that supports converting between string, stream, Json, and bytes.
+    /// </summary>
+    public class BinaryData
+    {
+        private const int CopyToBufferSize = 81920;
+
+        private static readonly UTF8Encoding s_encoding = new UTF8Encoding(false);
```

Comment:
should true be passed to the second parameter of this ctor?

Reply:
As mentioned here - https://github.com/Azure/azure-sdk-for-net/pull/16248#discussion_r512099122, we can just remove this and use Encoding.Uft8.